repl/src/mescal/repl_utils.clj

```diff
@@ -20,7 +20,7 @@
   (cf/formatter "EEE MMM dd HH:mm:ss zzz yyyy"))
 
 (defn- parse-timestamp [timestamp]
-  (Timestamp. (.getMillis (cf/parse timestamp-formatter timestamp))))
+  (Timestamp. (.getMillis (cf/parse timestamp-formatter (string/replace timestamp #"  +" " ")))))
```

Comment:
The timestamp being displayed by `auth-tokens-refresh` contains multiple spaces between the month name and the day when the day has only one digit.